src/components/Loader.js

```diff
@@ -1,25 +1,41 @@
-import React from 'react'
+import React, { useMemo } from 'react'
+import { useRouteMatch } from 'react-router'
 import Lottie from 'react-lottie-player'
 import styled from 'styled-components'
+
 import beeAnimation from '@assets/lotties/bee-animation.json'
+import gardensLoader from '@assets/lotties/gardens-loader.json'
+import { HIVE_GARDEN_ADDRESS } from '@/constants'
 
 const Wrapper = styled.div`
   pointer-events: none;
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  height: 500px;
-  width: 100%;
+  position: fixed;
```

Comment:
We should use `absolute` instead.  `fixed` position is always relative to the viewport, doesn´t respect parent´s layout (e.g You can see that the loader for the list of gardens is off)

We would also need to add a parent div with a 100% height to the Garden Home and ProposalDetail which renders the Loader to properly center it in this case.